DocumentiPubblici/AnalisiDeiRequisiti/Sezioni/CasiUso/8DashboardVenditore.tex

```diff
@@ -40,8 +38,9 @@ \subsection{UC18 - Aggiunta Prodotto}
     \end{itemize}
 \end{itemize}
 
-\subsubsection{UC18.1 - Aggiunta o rimozione prodotto nella sezione dei prodotti in evidenza}
-\label{UC19.1}
+\resetSubUC
+
+\subUC{Aggiunta o rimozione prodotto nella sezione dei prodotti in evidenza}
```

Comment:
Questo deve essere un UC a sua volta in quanto permette in più di rimuovere

Reply:
Stessa cosa